src/Omega_h_hypercube.hpp

```diff
@@ -485,6 +488,10 @@ OMEGA_H_INLINE constexpr Int hypercube_split_degree(
   return hypercube_degree(parent_dim, parent_dim - split_dim);
 }
 
+void mark_amr(Mesh* mesh, Read<Byte> elem_mark);
```

Comment:
There should be a new file `Omega_h_amr_topology.hpp` (or something with AMR in the name), this stuff doesn't belong in the hypercube header.